playbooks/roles/edxapp/defaults/main.yml

```
@@ -1573,6 +1573,7 @@ edxapp_cms_variant: cms
 
 # Worker Settings
 worker_django_settings_module: '{{ EDXAPP_SETTINGS }}'
+worker_django_disable_heartbeats: true
```

At first glance, the use of a positive boolean `true` to convey disabling a feature seems a little awkard. Perhaps this could be flipped around:

```suggestion
worker_django_enable_heartbeats: false
```

---

I agree it seems a little awkward. I'll change this to `worker_django_enable_heartbeats` :grin: